sdk/servicebus/azure-servicebus/tests/perf_tests/_test_base.py

```diff
@@ -5,124 +5,210 @@
 
 import uuid
 
-from azure_devtools.perfstress_tests import PerfStressTest, get_random_bytes
+from azure_devtools.perfstress_tests import PerfStressTest, get_random_bytes, BatchPerfTest
 
-from azure.servicebus import ServiceBusClient, ServiceBusReceiveMode, ServiceBusMessage
+from azure.servicebus import ServiceBusClient, ServiceBusReceiveMode, ServiceBusMessage, TransportType
 from azure.servicebus.aio import ServiceBusClient as AsyncServiceBusClient
 from azure.servicebus.aio.management import ServiceBusAdministrationClient
 
-MAX_QUEUE_SIZE = 40960
+MAX_QUEUE_SIZE=4096
```

Comment:
Question: Why is this smaller is it b/c of the pyamqp message?

Reply:
good catch, I need to get rid of this. That value is in MB, so it was trying to create a queue of size 40 GB so I had to bump it down to a valid value. Since the tests are no longer creating/deleting queues dynamically, dont need it